core/os/os.h

```diff
@@ -287,6 +288,16 @@ class OS {
 	virtual bool is_in_low_processor_usage_mode() const;
 	virtual void set_low_processor_usage_mode_sleep_usec(int p_usec);
 	virtual int get_low_processor_usage_mode_sleep_usec() const;
+	virtual void set_update_vital_only(bool p_enabled);
+	virtual bool is_update_vital_only() const { return _update_vital_only; }
+
+	// Convenience easy switch for turning this off outside tools builds, without littering calling code
+	// with #ifdefs. It will also hopefully be compiled out in release.
+#ifdef TOOLS_ENABLED
+	bool is_update_vital_only_active() const { return is_update_vital_only(); }
+#else
+	bool is_update_vital_only_active() const { return false; }
+#endif
```

Comment:
You could put the `TOOLS_ENABLED` check directly in `is_update_vital_only` so you don't need the convenience wrapper.

Reply:
Ok will do. :+1:

Reply:
Done.